### PR TITLE
feat: add swap package (imported from `Emurgo/yoroi-dex`)

### PR DESCRIPTION
**Pull request by neuodev**
We will move the [`yoroi-dex`](https://github.com/Emurgo/yoroi-dex) to this repo for must faster development cycle. The plan is to remove or archive the `yoroi-dex` and start working on the `swap-react` package next.

### CODE REVIEW

**Review by abdelkrimdev — APPROVED** [review]

File: packages/swap/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@yoroi/swap",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "repository": "https://github.com/Emurgo/yoroi-dex.git",
```
Comment (stackchain) [inline packages/swap/package.json]:
```suggestion
  "repository": {
    "type": "github",
    "url": "https://github.com/Emurgo/yoroi.git",
    "directory": "packages/swap"
  },
```

File: packages/swap/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "author": "EMURGO",
+  "license": "MIT",
```
Comment (stackchain) [inline packages/swap/package.json]:
```suggestion
  "license": "Apache-2.0",
```

File: packages/swap/yarn.lock
Comment (stackchain) [inline packages/swap/yarn.lock:1]: it can be removed -> `yarn install` at root of project.

File: packages/swap/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@yoroi/swap",
+  "version": "0.0.1",
+  "main": "dist/index.js",
```
Comment (stackchain) [inline packages/swap/package.json:4]:
```suggestion
  "main": "dist/index.js",
  "source": "index",
```

File: packages/swap/.nvmrc [hunk 4]
```diff
@@ -0,0 +1 @@
+v16.15.0
```
Comment (stackchain) [inline packages/swap/.nvmrc]:
```suggestion
v16.19.0
```

**Review by stackchain — COMMENTED** [review]
@neuodev few suggestions.

**Pull request merged**